cdap-runtime-ext-dataproc/pom.xml

```diff
@@ -135,6 +135,14 @@
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
```

Comment:
Why do we need both Mockito and Powermock?

Reply:
Oh wow, we are using 8 year old Mockito version. Could you try bumping it to newer one, so we don't need powermock for static calls?